### PR TITLE
Update Calico Quickstart: master terminology replaced with control-plane

### PR DESCRIPTION
**Pull request by ebCrypto**
Product Version(s):

Issue:

Quickstart refers to master nodes even though they are deprecated and replaced with control-planes

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

### CODE REVIEW

**Comment by bmckercher123** [comment]
Good catch. We have an existing docs ticket to search and replace all of these occurrences (not just the one you flagged). I'll check to see if we want to merge this item, or pull this ticket up in priority to address all.

File: calico_versioned_docs/version-3.25/getting-started/kubernetes/quickstart.mdx [hunk 1]
```diff
@@ -128,13 +128,16 @@ The geeky details of what you get:
...
 
-1. Remove the taints on the master so that you can schedule pods on it.
-
+1. Remove the taints on the control-plane so that you can schedule pods on it. 
 
    ```bash
    kubectl taint nodes --all node-role.kubernetes.io/control-plane-
-   kubectl taint nodes --all node-role.kubernetes.io/master-
    ```
+   > With the 1.25 release of kubeadm the taint node-role.kubernetes.io/master is no longer applied to control plane nodes.
```
Comment (ctauchen) [inline calico_versioned_docs/version-3.25/getting-started/kubernetes/quickstart.mdx:136]: The intention here is to have a command that works for for multiple versions. If you've tested this on 1.25 -- what happens if you run `kubectl taint nodes --all node-role.kubernetes.io/master-`?

If there's an error, then we'll need to make changes. If this is just an unnecessary extra, we'll probably keep it in place until we stop supporting Kubernetes versions earlier than 1.25.

https://docs.tigera.io/calico/latest/getting-started/kubernetes/requirements#supported-versions

**Review by ctauchen — COMMENTED** [review]
Hi @ebCrypto, thanks for this! You've prompted me to return to the bug ticket @bmckercher123 mentioned above.

The PR is here: https://github.com/tigera/docs/pull/565, and it covers the terminology changes in your PR. We had a good few changes like these to make.

File: calico_versioned_docs/version-3.25/getting-started/kubernetes/quickstart.mdx (hunk 1 above)
Comment (ebCrypto) [reply calico_versioned_docs/version-3.25/getting-started/kubernetes/quickstart.mdx:136]:
```
$kubectl taint nodes --all node-role.kubernetes.io/master-
error: taint "node-role.kubernetes.io/master" not found
```
The above came from a 1.26 cluster
```
$kubectl get nodes -o wide
NAME    STATUS   ROLES           AGE   VERSION   INTERNAL-IP   EXTERNAL-IP   OS-IMAGE             KERNEL-VERSION      CONTAINER-RUNTIME
k8s04   Ready    control-plane   12m   v1.26.0   10.0.15.7     <none>        Ubuntu 22.04.2 LTS   5.15.0-69-generic   containerd://1.6.20
```

Comment (ctauchen) [reply calico_versioned_docs/version-3.25/getting-started/kubernetes/quickstart.mdx:136]: Ok, that's a problem then. We'll need to find another way to present this information.

I've opened a docs bug internally. We'll fix this here and in other versions.

**Comment by ctauchen** [comment]
Closing here, tracking elsewhere.

**Pull request closed without merging**